### DISCUSSION TITLE
Unexpected argument '-m' found

### DISCUSSION THREAD

**Discussion by markcancode3000** [foundry-rs/foundry#5268] [Help] [answered]
I've checked through the code and everything looks good to me. Every time I run the FundMeTest this error pops up: (error: unexpected argument '-m' found ). Did  I miss something when I ran anvil or is there some plug in I may be missing? Any clues would help, thanks yall!

![Screen Shot 2023-07-01 at 11 33 52 PM](https://github.com/foundry-rs/foundry/assets/95047407/b15e2ae3-671f-4102-b025-7ebc0bb33720)

**Comment by mds1** [comment, marked as answer]
`-m` has been deprecated and was replaced with `--match-test` / `--mt`

**Reply by markcancode3000** [reply]
Thank you!!!!

**Reply by tinteth** [reply]
thanks!

**Reply by smoke704** [reply]
Sweet, thanks!

**Reply by 0xJoichiro** [reply]
Thank you
